ArduPlane/Parameters.cpp

```diff
@@ -1422,6 +1422,21 @@ void Plane::load_parameters(void)
         }
     }
 
+    // Convert SOAR_ENABLE_CH to RCx_OPTION
+    AP_Int8 soar_enable_ch;
+    AP_Param::ConversionInfo soar_info = {
+        Parameters::k_param_g2,
+        968,
+        AP_PARAM_INT8,
+        nullptr
+    };
+    if (AP_Param::find_old_parameter(&soar_info, &soar_enable_ch) && soar_enable_ch.get() != 0) {
```

Comment:
Might as well make this ```soar_enable_ch.get() > 0``` for clarity, right?

Reply:
I followed the same pattern as the flap conversion code above.

Reply:
Fortunately that channel() method is pretty strict on its bounds-checking, so you can get away with this :-)